src/components/TodoHeader.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="flex justify-between">
+    <h1 class="text-7xl font-bold">To do List</h1>
+    <PlusCircleIcon @click="$emit('addTodo')" class="h-16 w-16 text-green-400 cursor-pointer" />
```

Comment:
Please use the correct attribute order! Here is an example: https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/attributes-order.md
But we will send you the "Vue Bible" :)